CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ HIGHLIGHTS:
 
 * qemu-agent variable is not anymore env. variable, instead it is 
   domain terraform variable
-## 0.4.3 (August 14, 2018)
+
+# 0.4.3 (August 14, 2018)
```

Comment:
I would make this fix with another dedicated PR, it's not relevant to this one. Plus it looks like other releases are still using a `h2` instead of a `h1`, it would be nice to be consistent within the whole file.

Reply:
thx flavio i just overlooked the additional `#` :smile:  i was thinking i added only a new-line :smile: 

i will change this